dohproxy/httpproxy.py

```diff
@@ -135,8 +145,19 @@ def main():
     args = parse_args()
     app = get_app(args)
 
-    ssl_context = setup_ssl(args)
+    # Get trusted reverse proxies and format it for aiohttp_remotes setup
+    if len(args.trusted) == 0:
+        x_forwarded_handling = aiohttp_remotes.XForwardedRelaxed()
+    else:
+        x_forwarded_handling = \
+            aiohttp_remotes.XForwardedStrict([args.trusted])
```

Comment:
should this just be:
```
aiohttp_remotes.XForwardedStrict(args.trusted)
```
no brackets?

Reply:
The XForwardedStrict seems to need the trusted hosts wrapped in an additional list.

See: https://github.com/aio-libs/aiohttp-remotes/blob/master/tests/test_x_forwarded.py#L100